ember/ember.d.ts

```diff
@@ -1765,9 +1788,43 @@ declare module Ember {
         toArray(): any[];
     }
     module RSVP {
+        interface IPromiseFunction {
+            (value: any):void;
+        }
+
+        interface IPromiseResolverFunction {
+            (resolve: IPromiseFunction, reject: IPromiseFunction): void;

```

Comment:
Maybe `PromiseResolve` and `PromiseReject` would be good interface names for these. I'd steer away from using the word `Function` in an interface.